WUM/wum-sce-test-wso2ei-6.2.0-full-testgrid.yaml

```diff
@@ -0,0 +1,49 @@
+# TestGrid job configuration
+
+version: '0.9'
+emailToList: "builder@wso2.org,kasung@wso2.com,chaminda@wso2.com,isuruu@wso2.com,nandika@wso2.com,
+              milindap@wso2.com,dulanjali@wso2.com,sasikala@wso2.com,isuruuy@wso2.com,chalitha@wso2.com"
+infrastructureConfig:
+  iacProvider: CLOUDFORMATION
+  infrastructureProvider: AWS
+  containerOrchestrationEngine: None
+  includes:
+  - Ubuntu-18.04
+  - CentOS-7.5
+  - MySQL-5.6
+  - MySQL-5.7
+  - Oracle-SE2-12.1
+  - SQLServer-SE-13.00
+  - SQLServer-SE-14.00
+  - Postgres-9.6
+  - Postgres-10.5
```

Comment:
For daily wum builds we do not require all these combinations. It's true that Testgrid supports all these, but running these daily is not cost-effective. We should rather run all these monthly/quarterly.

See the list of infras we recommend:

https://github.com/wso2/testgrid-job-configs/blob/0c3fec28972defe36d417cdbf2a4db65a07e5508/WUM/wum-sce-test-wso2ei-6.4.0-full-testgrid.yaml#L10-L16

Reply:
Added recommended infras.